src/lib/swapper/swappers/LifiSwapper/utils/getIntermediaryTransactionOutputs/getIntermediaryTransactionOutputs.ts

```diff
@@ -28,6 +28,7 @@ export const getIntermediaryTransactionOutputs = (
       const assetId = lifiTokenToAssetId(step.action.toToken)
 
       const asset = assets[assetId] ?? {
+        assetId,
         chainId: lifiChainIdToChainId(step.action.toToken.chainId),
```

Comment:
Can use `makeAsset()` if you feel like it - this will make `chainId` redundant since it will be deserialized from `assetId`

Reply:
potato potato, done :)